server/storage/mvcc/watchable_store.go

```diff
@@ -604,16 +602,6 @@ func (w *watcher) send(wr WatchResponse) bool {
 		wr.Events = ne
 	}
 
-	verify.Verify(func() {
-		if w.startRev > 0 {
-			for _, ev := range wr.Events {
-				if ev.Kv.ModRevision < w.startRev {
-					panic(fmt.Sprintf("Event.ModRevision(%d) is less than the w.startRev(%d) for watchID: %d", ev.Kv.ModRevision, w.startRev, w.id))
-				}
-			}
-		}
-	})
-
```

Comment:
I strongly suggest to keep this commit in your test. So that we can exclude the server side issue. You also need to keep the field `startRev` in `watcher`

Reply:
agree